nms/app/packages/magmalte/app/components/FEGServicingAccessGatewayKPIs.js

```diff
@@ -37,7 +37,7 @@ import {useRouter} from '@fbcnms/ui/hooks';
  */
 export async function getServicedAccessNetworks(
   federationNetworkId: network_id,
-  enqueueSnackbar: (
+  enqueueSnackbar?: (
```

Comment:
Made this optional because we need to call this function from the useRefreshingContext and in that function enqueueSnackbar is optional which forces us to adhere to that. I think the enqueueSnackbar in useRefreshingContext was made optional so as to don't make errors noisy sometimes.